runtime/centrifuge/src/lib.rs

```diff
@@ -1065,6 +1614,19 @@ construct_runtime!(
 		OrmlTokens: orml_tokens::{Pallet, Storage, Event<T>, Config<T>} = 151,
 		OrmlAssetRegistry: orml_asset_registry::{Pallet, Storage, Call, Event<T>, Config<T>} = 152,
 		OrmlXcm: orml_xcm::{Pallet, Storage, Call, Event<T>} = 153,
+
+		// Synced pallets across all runtimes - Range: 180-240
+		// WHY: * integrations like fireblocks will need to know the index in the enum
+		//      * makes it easier, without parsing complete metadata
+		//      * makes it in-sync for XCM integrations -- same enum variant again
+		PoolRegistry: pallet_pool_registry::{Pallet, Call, Storage, Event<T>} = 180,
+		PoolSystem: pallet_pool_system::{Pallet, Call, Storage, Event<T>} = 181,
+		Permissions: pallet_permissions::{Pallet, Call, Storage, Event<T>} = 182,
+		Investments: pallet_investments::{Pallet, Call, Storage, Event<T>} = 183,
+		InterestAccrual: pallet_interest_accrual::{Pallet, Storage, Event<T>, Config<T>} = 184,
```

Comment:
`pallet_interest_accrual` sets the `StorageVersion` to `Release::v2` in its `GenesisBuild`. However, we don't run any migration for Centrifuge. As a result, the `StorageVersion` storage will default to `0`. This needs to be addressed. 

Additionally, `pallet_interest_accrual` still uses the old scheme of handling `StorageVersion`, but this is another story. With the "new" way, the pallet would actually be initialized with the latest version for Centrifuge runtime.

Reply:
That was a very good catch!
By now, can it be fixed applying the migration as it is done in `altair`? 
cc @branan

Reply:
All we need to do is set `pallet_interest_accrual::StorageVersion` to `ReleaseV2`. As there is no storage trie yet on the Centrifuge runtime, no other migration is required.

Theoretically, we could also handle this the next time we bump the version but let's not start such inconsistencies.